zeebe/engine/src/main/java/io/camunda/zeebe/engine/processing/job/DefaultJobCommandPreconditionGuard.java

```diff
@@ -32,14 +32,16 @@ public DefaultJobCommandPreconditionGuard(
   }
 
   public boolean onCommand(
-      final TypedRecord<JobRecord> command, final CommandControl<JobRecord> commandControl) {
+      final TypedRecord<JobRecord> command,
```

Comment:
Frankly, I'd like them to follow the same pattern as the other processors instead of the attempt at making it easier using the `CommandProcessor` interface. But yeah, I didn't want to refactor this as part of this PR.